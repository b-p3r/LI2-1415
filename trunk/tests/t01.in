3 3
SIA
ORR
EDI
