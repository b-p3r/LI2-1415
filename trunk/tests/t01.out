A
AI
AIO
AIS
AR
ARDE
ARDER
ARDOR
ARE
ARIDO
ARIDOS
ARO
AROS
ARRE
ARREDO
DE
DER
DERA
DIRA
DIRIA
DO
DOE
DOER
DOI
DOIA
DOIRAR
DOIS
DOR
DOS
E
ERA
ERARIO
ERRA
IA
IDE
IDO
IDOS
IR
IRA
IRE
IRIA
IRIS
IRRA
IS
O
ODE
ODER
ODRE
OE
ORA
ORAI
ORAIS
ORAR
ORE
OS
RA
RAIO
RAIOS
RARO
RAROS
RE
RI
RIA
RIDE
RIDO
RIO
RIOS
RIR
RIRA
RIRIA
RIS
RISO
RO
RODE
ROI
ROIA
SI
SIR
SO
SOE
SOR
SORRI
SORRIA
